### PR TITLE
Fixes top level navigation bug that showed "Teach" twice, also style fixes and css cleanup

### PR DESCRIPTION
**Pull request by jtamiace**
Pushing these changes now so that it's easier to work with making the second level navigation under teach and manage responsive. 

Reference this issue:
https://github.com/learningequality/ka-lite/issues/3168#issuecomment-76888913

### CODE REVIEW

File: kalite/control_panel/templates/control_panel/data_export.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% load staticfiles %}
 {% load include_block %}
 
-{% block facility_active %}facility-active active-tab active-nav{% endblock facility_active %}
+{% block facility_active %}active{% endblock facility_active %}
```
Comment (MCGallaspy) [inline kalite/control_panel/templates/control_panel/data_export.html]: Can you add `{% block i18n_do_not_translate %}` and `{% endblock i18n_do_not_translate %}` tags surrounding this section?

File: kalite/control_panel/templates/control_panel/facility_management.html [hunk 2]
```diff
@@ -4,8 +4,7 @@
 {% load staticfiles %}
 
 {% block control_panel_active %}{% endblock %}
-{% block users_active %}active{% endblock %}
-{% block user_active %}users-active active-tab active-nav{% endblock user_active %}
+{% block user_active %}active{% endblock user_active %}
```
Comment (MCGallaspy) [inline kalite/control_panel/templates/control_panel/facility_management.html]: And add `{% block i18n_do_not_translate %}` tags around here, too.

File: kalite/control_panel/templates/control_panel/zone_management.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 {% block users_active %}active{% endblock users_active %}
 {% block manage_active %}active{% endblock manage_active %}
-{% block facility_active %}facility-active active-tab active-nav{% endblock facility_active %}
+{% block facility_active %}active{% endblock facility_active %}
```
Comment (MCGallaspy) [inline kalite/control_panel/templates/control_panel/zone_management.html]: And here.

File: kalite/playlist/templates/playlist/assign_playlists.html [hunk 4]
```diff
@@ -3,11 +3,12 @@
 {% load i18n %}
 {% load kalite_staticfiles %}
 
+
 {% block home_active %}active{% endblock home_active %}
 {% block playlist_active %}playlists-active active-tab active-nav{% endblock playlist_active %}
 {% block teacher_active %}active{% endblock teacher_active %}
-{% block admin_active %}active{% endblock admin_active %}
 {% block assign_playlists_active %}active{% endblock assign_playlists_active %}
```
Comment (MCGallaspy) [inline kalite/playlist/templates/playlist/assign_playlists.html]: And put `{% block i18n_do_not_translate %}` tags around here, too.

**Comment by MCGallaspy** [comment]
Just noticed some places where we should put `{% block i18n_do_not_translate %}` tags, so that the css classes aren't inadvertently translated.

File: kalite/control_panel/templates/control_panel/data_export.html [hunk 5]
```diff
@@ -3,7 +3,9 @@
 {% load staticfiles %}
 {% load include_block %}
 
-{% block facility_active %}facility-active active-tab active-nav{% endblock facility_active %}
+{% block i18n_do_not_translate %}
+{% block facility_active %}active{% endblock facility_active %}
```
Comment (aronasorman) [inline kalite/control_panel/templates/control_panel/data_export.html]: Indent if inside the `i18n_do_not_translate` block.

Comment (aronasorman) [inline kalite/control_panel/templates/control_panel/data_export.html]: Do the same for the other files as well.

File: kalite/distributed/static/css/distributed/khan-lite.css [hunk 6]
```diff
@@ -315,44 +276,26 @@ a:hover {
 .manage-nav span.icon {
     font-size: 40pt;
 }
+/*.top-level-active {
```
Comment (aronasorman) [inline kalite/distributed/static/css/distributed/khan-lite.css]: Delete commented out code.

**Comment by MCGallaspy** [comment]
Groovy.

**Pull request merged**